### PR TITLE
Slack Integration update

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/error-reporting/workflow-integrations/messaging/slack.md [hunk 1]
```diff
@@ -9,25 +9,55 @@ import Tabs from '@theme/Tabs';
...
-An Incoming Webhook must be configured in Slack. To set up the Incoming Webhook, follow the on-screen instructions on the [Incoming WebHooks](https://my.slack.com/services/new/incoming-webhook/) page:
+- Use the Slack App (recommended)
+- Connecting a Webhook (legacy)
 
-1. In the **Post to Channel** section, select the channel you want to post messages to, then select **Add Incoming WebHooks integration**.
-1. After the Incoming Webhook integration is added, from the **Setup Instructions**, copy the **Webhook URL**. The Webhook URL is used to create the integration for Slack in Backtrace.
+:::note
 
-## Create the Integration
+Slack announced that they plan to deprecated support for customized webhooks, so the App for Slack is generally preferring.
```
Comment (peter-shao) [inline docs/error-reporting/workflow-integrations/messaging/slack.md:21]: deprecate*

File: docs/error-reporting/workflow-integrations/messaging/slack.md [hunk 2]
```diff
@@ -9,25 +9,55 @@ import Tabs from '@theme/Tabs';
...
+
+## Main Body Content​
+
+You can specify attributes to be appended to the description in the Main body content.
+
+<img src={useBaseUrl('/img/error-reporting/workflow-integrations/slack/body-content.png')} alt="main body content"/>
+
+## Issue Based Alerts​
+
+To further automate your workflow, you can configure automated actions for your Jira workflow integration with issue-based alerts. Use issue-based alerts to automatically generate issues in Jira based on the conditions and frequency that you specify. For more information, see [Issue Based Alerts](/error-reporting/project-setup/alerts/#issue-based-alerts).
```
Comment (peter-shao) [inline docs/error-reporting/workflow-integrations/messaging/slack.md:63]: Jira -> Slack

File: docs/error-reporting/workflow-integrations/messaging/slack.md (hunk 1 above)
Comment (LoryApiFortress) [inline docs/error-reporting/workflow-integrations/messaging/slack.md:21]:
```suggestion
Slack announced that they plan to deprecate support for customized webhooks, so the App for Slack is generally preferred.
```

File: docs/error-reporting/workflow-integrations/messaging/slack.md (hunk 2 above)
Comment (LoryApiFortress) [inline docs/error-reporting/workflow-integrations/messaging/slack.md:63]:
```suggestion
You can configure automated actions for your Slack workflow integration with issue-based alerts to further automate your workflow. Use issue-based alerts to automatically generate issues in Slack based on the conditions and frequency you specify. For more information, see [Issue Based Alerts](/error-reporting/project-setup/alerts/#issue-based-alerts).
```

**Pull request merged**